atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -0,0 +1,278 @@
+        cluster = self.weaviate_client.get_embeddings_by_property(CollectionNames.CLUSTERCENTER.value, "cluster_id", competency["properties"]["cluster_id"])
+
+        new_text_competencyID = text["properties"]["competency_ids"]
+        new_text_competencyID.append(competency_id)
+
+        new_text = { "properties": [{
+                    "text_id":  text["properties"]["text_id"],
+                    "text":  text["properties"]["text"] ,
+                    "competency_ids": new_text_competencyID
+        }]}
+
+        self.weaviate_client.update_property_by_id(CollectionNames.TEXT.value, text["properties"]["text_id"], new_text)
```

Ruff (0.11.9)

Line too long (119 > 88)

(E501)